chirp/drivers/ga510.py

```diff
@@ -1074,12 +1084,6 @@ class TDH6Radio(RadioddityGA510Radio):
     VENDOR = "TIDRADIO"
     MODEL = "TD-H6"
 
-    def get_features(self):
-        rf = super().get_features()
-        rf.valid_bands = [(136000000, 174000000),
-                          (400000000, 520000000)]
-        return rf
-
```

Comment:
Why are you removing this? It was *just* added recently and removing it will regress that bug.